src/components/TextEditor.vue

```diff
@@ -187,29 +216,30 @@ export default {
 				editor.editing.view.focus()
 			}
 
-			// Set value as late as possible, so the custom schema listener is used
-			// for the initial editor model
-			this.text = this.sanitizedValue
-
-			logger.debug(`setting TextEditor contents to <${this.text}>`)
-
 			this.bus.$on('append-to-body-at-cursor', this.appendToBodyAtCursor)
-			this.bus.$on('insert-signature', this.onInsertSignature)
+			this.$emit('ready', editor)
 		},
-		onInput() {
-			logger.debug(`TextEditor input changed to <${this.text}>`)
-			this.$emit('input', this.text)
+		onEditorInput(text) {
+			text = this.sanitizeValue(text)
```

Comment:
Inconvenient to call DomPurify here again ...

Reply:
Resolved via https://github.com/nextcloud/mail/commit/ea90bc1f4c668592e9d7d0ee1dfdf787077cd60b